### PR TITLE
Make update field generic

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Sessions.Data/SessionsRepository.fs [hunk 1]
```diff
@@ -1,24 +1,17 @@
...
     let guid = Guid.NewGuid()
-    insert {session with Id = guid; Date = Nullable DateTime.UtcNow;} |> ignore
+    insert { session with Id = guid ; Date = Nullable DateTime.UtcNow } |> ignore
     guid
 
-let getAll () = getConnection().GetAll<Session>()
-
-let get (sessionId : Guid) = getConnection().Get<Session>(sessionId)
+let getAll() = getConnection().GetAll<Session>()
+let get (sessionId : Guid) = getConnection().Get<Session> (sessionId)
```
Comment (pallenSL) [inline Sessions.Data/SessionsRepository.fs]: Newline please!

File: Sessions/ProfilesController.fs [hunk 2]
```diff
@@ -14,6 +14,7 @@ type ProfilesController() =
     let patch (id: Guid) (op: PatchOp) =
         match op.Path with 
         | "rating" -> 
+            //TODO not protecting against an invalid rating? Did Chris say do this in the database?
```
Comment (pallenSL) [inline Sessions/ProfilesController.fs]: Stick it in Jira, delete from here

File: Sessions.Data/ProfilesRepository.fs [hunk 3]
```diff
@@ -1,25 +1,17 @@
...
+let add (profile : Profile) = 
     let guid = Guid.NewGuid()
     insert { profile with Id = guid } |> ignore
     guid
 
-let getAll () = getConnection().GetAll<Profile>()
-
-let get (profileId : Guid) = getConnection().Get<Profile>(profileId)
+let getAll() = getConnection().GetAll<Profile>()
+let get (profileId : Guid) = getConnection().Get<Profile> (profileId)
```
Comment (pallenSL) [inline Sessions.Data/ProfilesRepository.fs]: I'm sure we can spare a byte of disk space for a \n between these!

**Comment by jebbinSCL** [comment]
Adjusted

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**